src/_posts/platform/getting-started/2000-01-01-getting-started-with-modsecurity.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Getting started with ModSecurity on Scalingo
+modified_at: 2022-07-25 00:00:00
+tags: tutorial security firewall modsecurity apache nginx waf
+index: 11
+---
+
+ModSecurity is an open-source Web Application Firewall (WAF) that allows users
+to monitor, log and filter HTTP requests. A very common use case is to rely on
+a set of open-source rules provided by [OWASP](https://owasp.org/) (called
+[OWASP ModSecurity Core Rule Set (CRS)](https://owasp.org/www-project-modsecurity-core-rule-set/))
+to protect a web application against generic classes of vulnerabilities and
+Layer 7 attacks such as SQL injections (SQLi), Cross Site Scripting (XSS), ...
```

Comment:
nitpick: I wouldn't mention these acronyms as they are not used throughout the document.
```suggestion
Layer 7 attacks such as SQL injections, Cross Site Scripting...
```